### DISCUSSION TITLE
ESPhome Gerät auf anderen HA umziehen?

### DISCUSSION THREAD

**Discussion by linuxdep** [syssi/esphome-jk-bms#181] [General]
Hallo,
da ja das esphome-jk-bms auf einem x86 übersetzt werden muss, was ich auf einer Testumgebung in einer VM gemacht habe, aber wie bekomme ich das ESPhome Gerät jetzt auf die HA Umgebung auf dem Pi4 umgezogen?

**Comment by syssi** [comment]
Im ESPHome Addon wirst du dir YAML-Konfiguration erst bauen können, sobald der Build-Stack aarch64 kompatibel ist. Hinzufügen kannst du die Node als ESPHome-Gerät trotzdem:

```
Einstellungen -> Geräte -> Gerät hinzufügen -> ESPhome -> Hostname oder IP benennen
```
